codersdk/features.go

```diff
@@ -27,3 +33,21 @@ type Entitlements struct {
 	Warnings   []string           `json:"warnings"`
 	HasLicense bool               `json:"has_license"`
 }
+
+type EntitlementsRequest struct {
+	// placeholder so that we can add request parameters in future
+	// without breaking changes to the go API
+}
```

Comment:
I'm tempted not to do this unless we have certainty that all parameters would be optional.

Otherwise a syntax error wouldn't occur, but the API request may fail.

Reply:
We don't have certainty that all parameters will be optional, but adding a new non-optional parameter is always an API breaking change.

If we formulate things with this placeholder, we at least have a straightforward way to add optional parameters in future without a breaking change.  It's true that we could do `opts ...EntilementsRequestOptions` in future, but that's inconsistent with how other methods work, so I added the placeholder...

Reply:
Hmm, well, looks like my hand is being forced to what you suggest anyway.  Typescript doesn't like the empty struct from our autogenerated code, so I'll just drop this.  Not worth fixing for something we're not crazy about in the first place.